Classes/Domain/Repository/ContentRepository.php

```diff
@@ -766,4 +771,30 @@ protected function getLanguageValidator() {
 		return GeneralUtility::makeInstance('TYPO3\CMS\Vidi\Domain\Validator\LanguageValidator');
 	}
 
+	/**
+	 * Signal that is called for post-processing the computed constraints object.
+	 *
+	 * @param Query $query
+	 * @param ConstraintInterface|NULL $constraints
+	 * @signal
+	 */
+	protected function emitPostProcessComputedConstraintsSignal(Query &$query, &$constraints) {
+		$this->getSignalSlotDispatcher()->dispatch(
+			'TYPO3\CMS\Vidi\Domain\Repository\ContentRepository',
+			'postProcessComputedConstraintsObject',
+			array(
+				&$query,
+				&$constraints,
```

Comment:
I am not sure why it seems necessary to use "&" for $query and $constraints here. AFAIK objects should automatically be passed by reference in PHP. But without those "&" it will not work.